SPECS/apr-util/apr-util.spec

```
@@ -1,25 +1,25 @@
-Summary:    The Apache Portable Runtime Utility Library
-Name:       apr-util
-Version:    1.6.1
-Release:        4%{?dist}
-License:    Apache License 2.0
-URL:        https://apr.apache.org/
-Group:      System Environment/Libraries
+Summary:        The Apache Portable Runtime Utility Library
+Name:           apr-util
+Version:        1.6.3
+Release:        1%{?dist}
+License:        Apache License 2.0
```

Since we're already here and we've recently switched to trying to use [SDPX identifiers](https://spdx.org/licenses/):


```suggestion
License:        Apache-2.0
```

---

Done :)